components/home/main/News.tsx

```diff
@@ -45,146 +45,157 @@ export default function News() {
       <p className="mt-3 mb-6 text-center text-base text-gray font-normal">
         {t('hm-p-news-text')}
       </p>
-      <div className="container mx-auto flex flex-wrap h-96">
+
+      <div className="grid grid-cols-1 md:grid-rows-2 lg:grid-rows-1 lg:grid-cols-3 gap-4">
```

Comment:
I would suggest that you add a max-width for this section, otherwise it looks bad on 2k and 4k resolutions